torch/csrc/jit/script/module.h

```diff
@@ -192,7 +192,7 @@ struct Method {
     size_t num_inputs = method.num_inputs();
     for(size_t i = 0; i < num_inputs; ++i) {
       const Value* v = g.inputs().at(i);
-      std::string name = v->hasUniqueName() ? v->uniqueName() : ("argument_"  + std::to_string(i));
+      std::string name = v->hasUniqueName() ? v->uniqueNameBase() : ("argument_"  + std::to_string(i));
```

Comment:
Why?

Reply:
Makes sense! I wasn't aware this part is for schema creation (the diff didn't include enough context).

Reply:
How hard would it be to enforce our uniqueNames are valid identifiers? We are about to start serializing by writing python-like functions to represent IR and it would be much more pleasant if we didn't let you write a.b in a unique name. That way this would be a non-issue.

Reply:
Maybe not that hard, let's see what breaks: https://github.com/pytorch/pytorch/pull/13501